libs/types/src/theme.ts

```diff
@@ -0,0 +1,73 @@
+import type { FlattenSimpleInterpolation } from 'styled-components/macro'
+
+export type CowSwapTheme = 'dark' | 'light' | 'darkHalloween' | 'lightChristmas' | 'darkChristmas'
+
+type ColorValue = string
+
+export interface Colors {
```

Comment:
This file is supposed to be a part of `libs/ui`, why it was relocated?

Reply:
Addressed now